ui/app/AppLayouts/Communities/views/EditAirdropView.qml

```diff
@@ -45,6 +45,10 @@ StatusScrollView {
     // Bool property indicating whether the fees are available
     required property bool feesAvailable
 
+    property string enabledChainIds
+    property string networkThatIsNotActive
+    property int networkIdThatIsNotActive
```

Comment:
Same here. It's an internal implementation detail, no need to be part of the component api.